SparkPost/2web2ui
# SA-1998: Added Form tracking component to industry benchmark modal

### What Changed

- Added `<Form />` component to industry benchmark modal

### How To Test

- Add Inbox Folder Rate as metric in report builder
- Click Industry Benchmark above chart
- Verify that the industry benchmark form still works

### To Do
